files.php

```diff
@@ -256,12 +257,12 @@ function find_genome($path){
 					$genome = '';
 					// Find genome from BAM or SAM files
 					if(substr($path, -4) == '.bam' || substr($path, -4) == '.bam'){
-						$bam_header = shell_exec (escapeshellcmd ('samtools view -H '.$path));
+						$bam_header = shell_exec (escapeshellcmd ('/storage/Software/packages/samtools-1.3.1/bin/samtools view -H '.$path));
```

Comment:
This will break Labrador for other users. Is there a reason that samtools isn't on the `PATH` for you? If you really need to specify path, it'd be better to use an optional config variable instead which defaults to an empty string..